gravitee-apim-console-webui/src/management/application/details/general/general-ng/application-general-ng.component.html

```diff
@@ -28,7 +28,7 @@
 
           <mat-form-field class="details-card__header__info-inputs__first-row__type-field">
             <mat-label>Type</mat-label>
-            <input formControlName="type" matInput type="text" />
```

Comment:
What about juste remove this field. this information is already into `details-card__header__right-coll__info` right ?

Reply:
Done